config/factions/castle.json

```diff
@@ -147,8 +147,7 @@
 			"horde" : [ 2, -1 ],
 			"mageGuild" : 4,
 			"warMachine" : "ballista",
-			"moatDamage" : 70,
-			"moatHexes" : [ 11, 28, 44, 61, 77, 111, 129, 146, 164, 181 ],
+			"moatAbility" : "core:spell.castleMoat",
```

Comment:
@IvanSavenko we do not need to define such spell. If mod need one standard moat, it can use predefined moat. I changed spells only for having names exactly as H3.

Reply:
@IvanSavenko you can see, than conflux moat are using castle spell, and mods can do this too.